include/runtime/header.h

```diff
@@ -442,4 +461,16 @@ void sfprintf(writer *file, char const *fmt, Args &&...args) {
   }
 }
 
+template <typename... Args>
+void sfwrite(void const *ptr, size_t size, size_t nmemb, writer *file) {
```

Comment:
What `nmemb` stands for?

Reply:
Number of Memory Bytes?

Reply:
Number of members. As you can see, if `file` is a `FILE *`, it simply calls `fwrite`, which expects this signature.